### DISCUSSION TITLE
Training SNN on speech data

### DISCUSSION THREAD

**Discussion by RMalikM** [jeshraghian/snntorch#175] [General]
Hi, I want to use SNN for speech emotion recognition task. can somebody suggest how to encode speech data and train a SNN model to classify the audio into emotions class? 

@jeshraghian

**Comment by ridgerchu** [comment]
Hi,
There are some audio datasets encoded as the neuromorphic format for spiking neural network. They are not related to the emotion class, but I think those datasets maybe could help you :) You can check them by this link: https://tonic.readthedocs.io/en/latest/reference/generated/tonic.datasets.SHD.html
https://tonic.readthedocs.io/en/latest/reference/generated/tonic.datasets.SSC.html
Those links are from the tonic package, which is also capable with snnTorch.
